opentracing/harness/api_check.py

```diff
@@ -39,12 +40,28 @@ def check_baggage_values(self):
         """If true, the test will validate Baggage items by storing and
         retrieving them from the trace context. If false, it will only attempt
         to store and retrieve the Baggage items to check the API compliance,
-        but not actually validate stored values.  The latter mode is only
+        but not actually validate stored values. The latter mode is only
         useful for no-op tracer.
         """
         return True
 
+    def check_scope_manager(self):
+        """If true, the test suite will validate the `ScopeManager` propagation
+        to ensure correct parenting. If false, it will only use the API without
+        asserting. The latter mode is only useful for no-op tracer.
+        """
+        return True
+
+    def is_parent(self, parent, span):
```

Comment:
don't know if it's really needed, but it would be great that tests available in the `api_check.py` tests also the parenting so that a `ScopeManager` can be tested automatically when a new `Tracer` implementation is created.